classes/class.ilMumieTaskUserListGUI.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+/**
+ * MumieTask plugin
+ *
+ * @copyright   2019 integral-learning GmbH (https://www.integral-learning.de/)
+ * @author      Tobias Goltz (tobias.goltz@integral-learning.de)
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+/**
+ * This form is used to add, edit and validate MUMIE Server configurations
+ *
+ */
+class ilMumieTaskUserListGUI extends ilTable2GUI
```

Comment:
We need a more descriptive name. This class is not just a simple list of users. Something like `MumieTaskGradeOverviewGUI`

Reply:
done